liquibase-standard/src/main/java/liquibase/command/core/helpers/DatabaseChangelogCommandStep.java

```diff
@@ -124,10 +128,25 @@ public void cleanUp(CommandResultsBuilder resultsBuilder) {
      * Add java property arguments to changelog parameters
      * @param changeLogParameters the changelog parameters to update
      */
-    public void addJavaProperties(ChangeLogParameters changeLogParameters) {
+    private void addJavaProperties(ChangeLogParameters changeLogParameters) {
```

Comment:
Will changing this from `public` to `private` represent an API-breaking change?

Reply:
Should not break anything.. but I rolled back the change so we don't miss the focus here :)